test/functional/driver_default_nonstandard_test.rb

```diff
@@ -4,7 +4,7 @@
 class DriverNonstandardTest < MiniTest::Test
   def setup
     @driver = Driver.new
-    @events = Driver.state_machine.events
+    @events = Driver.state_machine(:status).events
```

Comment:
Why do you need to pass the name now?

Reply:
Without it this call would create a new state machine on `state`.

Any call to state_machine when not using the default name (`:state`) needs to also supply the namespace. I could update to always return a state machine if there is one? But though it would be better to keep it simple and consistent.

Can also amend the read me to depending on which way you decide to go.